### PR TITLE
Added version file and updated the respective files[V3]

### PR DESCRIPTION
**Pull request by sathnaga**
Added version file and updated the respective files

Signed-off-by: Satheesh Rajendran <sathnaga@linux.vnet.ibm.com>
Signed-off-by: Lukáš Doktor <ldoktor@redhat.com>

### CODE REVIEW

**Comment by sathnaga** [comment]
```
#cat VERSION 
41.0
# python setup.py install
running install
running bdist_egg
running egg_info
writing avocado_plugins_vt.egg-info/PKG-INFO
...
...
Installed /usr/local/lib/python2.7/dist-packages/avocado_plugins_vt-41.0-py2.7.egg
Processing dependencies for avocado-plugins-vt==41.0
Finished processing dependencies for avocado-plugins-vt==41.0


#cat VERSION 
41.0
# python setup.py install
running install
running bdist_egg
running egg_info
writing avocado_plugins_vt.egg-info/PKG-INFO
...
...
Adding avocado-plugins-vt 42.0 to easy-install.pth file

Installed /usr/local/lib/python2.7/dist-packages/avocado_plugins_vt-42.0-py2.7.egg
Processing dependencies for avocado-plugins-vt==42.0
Finished processing dependencies for avocado-plugins-vt==42.0


```

**Comment by sathnaga** [comment]
@ldoktor @clebergnu FYI

File: avocado-plugins-vt.spec [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 Summary: Avocado Virt Test Plugin
 Name: avocado-plugins-vt
-Version: 41.0
+Version: %{version}
```
Comment (ldoktor) [inline avocado-plugins-vt.spec]: This does not work (at least on my system). I tried to look for a way to read file or execute shell to get some value, but never found a way to do that. Anyway I know nothing about RPMs, so there might be a way, but I think we just have to update the VERSION and spec file manually.

Anyway the `make rpm` fails on my system so I think you should either find a way to get the version from the `VERSION` file or keep the version string here.

Comment (sathnaga) [inline avocado-plugins-vt.spec]: @ldoktor I tried aswell, `make rpm` fails at my env as well, now have reverted spec file change in this pull.

**Comment by ldoktor** [comment]
Thank you, @sathnaga, do you intend to send similar patches to other avocado-related projects, or should I take care of them?

**Comment by sathnaga** [comment]
@ldoktor sure will send it.

**Comment by sathnaga** [comment]
@ldoktor https://github.com/avocado-framework/avocado/pull/1483

**Pull request merged**